lib/project_types/script/messages/messages.rb

```diff
@@ -64,10 +64,18 @@ module Messages
                                               "%{missing_keys}.",
           config_ui_missing_keys_error_help: "Add the keys and try again.",
 
+          config_ui_invalid_type_error_cause: "The UI configuration file %{filename} only accept "\
+                                              "one of the following type(s): %{valid_types}.",
```

Comment:
Nit: Since `type` is being renamed to `inputMode`, I think we should update the text here to match.